core/frontend/src/IModelApp.ts

```diff
@@ -201,6 +202,11 @@ export class IModelApp {
   /** Event raised just before the frontend IModelApp is to be shut down */
   public static readonly onBeforeShutdown = new BeEvent<() => void>();
 
+  /** Event raised after IModelApp is finished starting up.
+   * @internal
+   */
+  public static readonly onBeforeStartup = new BeEvent<() => void>();
```

Comment:
> Event raised ***after*** IModelApp is finished starting up.
> on***Before***Startup

It's `@internal` so ignore if you like, but it should be called on***After***Startup.

Reply:
Yea, good point. I'll switch it to onAfterStartup